packages/core/admin/admin/src/components/PluginsInitializer/index.js

```diff
@@ -35,7 +35,12 @@ const PluginsInitializer = () => {
     );
   }
 
-  return <Admin plugins={plugins} />;
+  return (
+    <>
+      <ReleaseNotification />
```

Comment:
hmm, 🤔  I wonder why the release notification should be in the PluginsInitializer?

Reply:
Until we refactor the Admin page plus we will switch to a hook instead of a component.